### DISCUSSION TITLE
result oscilatopn

### DISCUSSION THREAD

**Discussion by joeh1242** [jomjol/AI-on-the-edge-device#3248] [General]
Hi,
I see some result oscillation +0,1m3, -0,1m3. 
![image](https://github.com/user-attachments/assets/939f874a-caf9-4de2-9bbb-14817404b89b)

general readings looks OK
![image](https://github.com/user-attachments/assets/0f10da03-0449-4eaf-86cc-bc2c5f3c6ddc)

the raw value  is quite stabile see attached file
[watermeter 23.9.2024.xlsx](https://github.com/user-attachments/files/17099982/watermeter.23.9.2024.xlsx)

my setting :

[water config.txt](https://github.com/user-attachments/files/17100136/water.config.txt)

any Idea?

Thank you

**Comment by SybexX** [comment]
There's nothing you can do about it with the version you're using(15.6.0 or 15.7.0?), maybe main.ExtendedResolution = true can improve the behavior somewhat.
This function is available in the current rolling https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-numberchangeratethreshold

https://github.com/jomjol/AI-on-the-edge-device/actions/runs/10909156464
